solr/test-framework/src/java/org/apache/solr/util/BadHdfsThreadsFilter.java

```diff
@@ -41,6 +41,10 @@ public boolean reject(Thread t) {
       return true;
     } else if (name.startsWith("nioEventLoopGroup")) {
       return true; // netty threads waiting for a web server stop confirm that won't happen
+    } else if (name.startsWith("globalEventExecutor")) { // HADOOP-11219 (Hadoop and Netty 4)
+      return true;
+    } else if (name.startsWith("Command processor")) { // HDFS-14997
```

Comment:
Why does this leak, it's a daemon thread? https://github.com/apache/hadoop/blob/03cfc852791c14fad39db4e5b14104a276c08e59/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/BPServiceActor.java#L1309

Reply:
I don't know I removed that line and tests seem to pass. Not sure why I was seeing that.